pkg/webui/styles/fonts.styl

```diff
@@ -0,0 +1,49 @@
+  src:  local('Source Sans Pro Regular'), local('SourceSansPro-Regular'), url('../assets/fonts/source-sans-pro-regular.eot?#iefix') format('embedded-opentype'), url('../assets/fonts/source-sans-pro-regular.woff2') format('woff2'), /* Super Modern Browsers */ url('../assets/fonts/source-sans-pro-regular.woff') format('woff'), /* Modern Browsers */ url('../assets/fonts/source-sans-pro-regular.ttf') format('truetype'), /* Safari, Android, iOS */ url('../assets/fonts/source-sans-pro-regular.svg#SourceSansPro') format('svg') /* Legacy iOS */
+/* source-sans-pro-600 */
+@font-face
+  font-family: 'Source Sans Pro'
+  font-style: normal
+  font-weight: 600
+  src: url('../assets/fonts/source-sans-pro-600.eot') /* IE9 Compat Modes */
+  src: local('Source Sans Pro SemiBold'), local('SourceSansPro-SemiBold'), url('../assets/fonts/source-sans-pro-600.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */ url('../assets/fonts/source-sans-pro-600.woff2') format('woff2'), /* Super Modern Browsers */ url('../assets/fonts/source-sans-pro-600.woff') format('woff'), /* Modern Browsers */ url('../assets/fonts/source-sans-pro-600.ttf') format('truetype'), /* Safari, Android, iOS */ url('../assets/fonts/source-sans-pro-600.svg#SourceSansPro') format('svg') /* Legacy iOS */
+/* source-sans-pro-700 */
+@font-face
+  font-family: 'Source Sans Pro'
+  font-style: normal
+  font-weight: 700
+  src: url('../assets/fonts/source-sans-pro-700.eot') /* IE9 Compat Modes */
+  src: local('Source Sans Pro Bold'), local('SourceSansPro-Bold'), url('../assets/fonts/source-sans-pro-700.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */ url('../assets/fonts/source-sans-pro-700.woff2') format('woff2'), /* Super Modern Browsers */ url('../assets/fonts/source-sans-pro-700.woff') format('woff'), /* Modern Browsers */ url('../assets/fonts/source-sans-pro-700.ttf') format('truetype'), /* Safari, Android, iOS */ url('../assets/fonts/source-sans-pro-700.svg#SourceSansPro') format('svg') /* Legacy iOS */
+/* ibm-plex-mono-regular */
+@font-face
+  font-family: 'IBM Plex Mono'
+  font-style: normal
+  font-weight: 400
+  src: url('../assets/fonts/ibm-plex-mono-regular.eot') /* IE9 Compat Modes */
+  src: local('IBM Plex Mono'), local('IBMPlexMono'), url('../assets/fonts/ibm-plex-mono-regular.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */ url('../assets/fonts/ibm-plex-mono-regular.woff2') format('woff2'), /* Super Modern Browsers */ url('../assets/fonts/ibm-plex-mono-regular.woff') format('woff'), /* Modern Browsers */ url('../assets/fonts/ibm-plex-mono-regular.ttf') format('truetype'), /* Safari, Android, iOS */ url('../assets/fonts/ibm-plex-mono-regular.svg#IBMPlexMono') format('svg') /* Legacy iOS */
+
+@font-face
+  font-family: 'Material Icons'
+  font-style: normal
+  font-weight: 400
+  src: url('https://fonts.gstatic.com/s/materialicons/v38/flUhRq6tzZclQEJ-Vdg-IuiaDsNcIhQ8tQ.woff2') format('woff2')
```

Comment:
You're still importing this remotely. The icon font needs to be a local asset as well.

Reply:
https://github.com/TheThingsNetwork/lorawan-stack/pull/1193/commits/27dcd31151ccb64a5ad89e45899ed19616fbbc08